core/src/mindustry/core/NetClient.java

```diff
@@ -50,6 +50,8 @@
     /** Last sent client snapshot ID. */
     private int lastSent;
 
+    public boolean paused = false;
```

Comment:
I'm concerned about the fact that there are now *4* different states to check for the pausing (some of which may be accurate, some may not): `netClient.paused`, `netServer.paused`, `state.isPaused()` and `state.is(State.paused)`. This is rather confusing, to say the least.

Reply:
well i wasn't sure where to put them and how to properly sync them all 🙈